sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -234,15 +234,12 @@ protected void postRegister() {
         reconciler.start();
         reconciler.reconcile(driver);
         if (suppressReviveManager == null) {
-            suppressReviveManager = new SuppressReviveManager(
-                    stateStore,
-                    configStore,
-                    driver,
-                    eventBus,
-                    getPlanManagers());
+            suppressReviveManager = new SuppressReviveManager(driver, stateStore, getPlanCoordinator());
         }
 
-        suppressReviveManager.start();
+        if (processingOffers.compareAndSet(false, true)) {
```

Comment:
May be worth a comment that this expects to be called by both register() and reregister(). Should also mention that we want to re-invoke reconcilation each time but that the offer processing loop doesn't need to be restarted.